src/compas/geometry/_core/tangent.py

```diff
@@ -0,0 +1,46 @@
+import math
+
+
+def tangent_points_to_circle_xy(circle, point):
+    """Calculates the tangent points on a circle in the XY plane.
```

Comment:
do i understand correctly that you compute the points on the circle that create tangent lines from the given point?

Reply:
yes